config/unicorn.rb

```diff
@@ -0,0 +1,23 @@
+# config/unicorn.rb
+worker_processes Integer(ENV["WEB_CONCURRENCY"] || 3)
+timeout 15
+preload_app true
+
+before_fork do |server, worker|
+  Signal.trap 'TERM' do
+    puts 'Unicorn master intercepting TERM and sending myself QUIT instead'
+    Process.kill 'QUIT', Process.pid
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.